centaur/src/it/scala/centaur/AbstractCentaurTestCaseSpec.scala

```diff
@@ -180,4 +184,14 @@ abstract class AbstractCentaurTestCaseSpec(cromwellBackends: List[String], cromw
     )
   }
 
+  /**
+   * Clean up temporary zip files created for Imports testing.
+   */
+  private def cleanUpImports(wfData: WorkflowData) = {
+    wfData.zippedImports match {
+      case Some(zipFile) => zipFile.delete(swallowIOExceptions = true)
+      case None => //
+    }
+  }
```

Comment:
TOL: Curious why this moved from `Test` to `AbstractCentaurTestCase`. Not challenging it, just curious...

Reply:
Just moved it closer to it's sole consumer 🙂